python/flyft/_flyft/mesh.cc

```diff
@@ -15,9 +15,14 @@ class MeshTrampoline : public Mesh
         PYBIND11_OVERRIDE_PURE(double, Mesh, area, i);
         }
         
+        double volume() const override 
+        {
+        PYBIND11_OVERLOAD_PURE_NAME(double, Mesh, "total_volume", volume, );
```

Comment:
How about we just call this `volume` (and remove the extra `,`:
```suggestion
        PYBIND11_OVERLOAD_PURE(double, Mesh, volume);
```